src/Docker/Image.php

```diff
@@ -58,6 +58,21 @@ abstract class Image
      */
     private $imageDigests;
 
+    /**
+     * @var int
+     */
+    protected $retryMinInterval = 100;
```

Comment:
není tohle zbytečně málo? začal bych třeba na vteřině.

Reply:
dal jsem tak kompropmis 500ms, timpadem se to na sekundu dostane pri druhym retry (predtim az po 4 retry), cili takto:

  | delay | attempt
-- | -- | --
  | 500 | 1
  | 1000 | 2
  | 2000 | 3
  | 4000 | 4
  | 8000 | 5
  | 16000 | 6
  | 32000 | 7
  | 60000 | 8
  | 60000 | 9
  | 60000 | 10
  | 60000 | 11
  | 60000 | 12
  | 60000 | 13
  | 60000 | 14
  | 60000 | 15
  | 60000 | 16
  | 60000 | 17
  | 60000 | 18
  | 60000 | 19
  | 60000 | 20
  | 843500 |  
total | 14.05833 | mins